pkg/cmd/tests/tests_run.go

```diff
@@ -43,36 +40,9 @@ const (
 	ginkgoOutputInterceptorModeNone = "none"
 )
 
-var suites = ginkgotest.TestSuites{
-	{
-		Name: "all",
-		Description: templates.LongDesc(`
-		Runs all tests.
-		`),
-		DefaultParallelism: 42,
-	},
-	{
-		Name: "scylla-operator/conformance/parallel",
-		Description: templates.LongDesc(`
-		Tests that ensure an Scylla Operator is working properly.
-		`),
-		LabelFilter:        fmt.Sprintf("!%s", framework.SerialLabelName),
-		DefaultParallelism: 42,
-	},
-	{
-		Name: "scylla-operator/conformance/serial",
-		Description: templates.LongDesc(`
-		Tests that ensure an Scylla Operator is working properly.
-		`),
-		LabelFilter:        fmt.Sprintf("%s", framework.SerialLabelName),
-		DefaultParallelism: 1,
-	},
-}
-
 type RunOptions struct {
 	genericclioptions.IOStreams
-	genericclioptions.ClientConfig
-	TestFrameworkOptions
+	*TestFrameworkOptions
```

Comment:
why changing the embedding to a pointer?

Reply:
It's effectively inheritance and the object is "required" - the pointer makes it effectively optional and forces callers to check for nil on something that suppose to always be there.

Reply:
changed